CHANGES.md

```diff
@@ -12,12 +12,17 @@ To be released.
 
 ### Behavioral changes
 
--  `BlockChain<T>.GetNonce()` became to count staged transactions too during nonce
-   computation. [[#270]]
+ -  `BlockChain<T>.GetNonce()` became to count staged transactions too during
+    nonce computation.  [[#270]]
+ -  `Swarm` became to send and receive multiple blocks with one message instead 
+    of one block per message.  [[#273], [#276]]
```

Comment:
How about this?

> A message `Swarm` makes became to have multiple blocks within it, which means round trips on the network are now much reduced.

Reply:
Also we should mention that an existing message type (`Block`) was gone and a new message type (`Blocks`) was introduced instead.

Reply:
Should this be a change of behavior?

> Also we should mention that an existing message type (`Block`) was gone and a new message type (`Blocks`) was introduced instead.

Should this be `Behavioral changes`? This is Backward-incompatible but it's not about the interface.

Reply:
It might be good to have a new section for protocol-level compatibility, but at the moment <q>behavior changes</q> seems the most appropriate section.